Realm/RLMUUID.mm

```diff
@@ -0,0 +1,45 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2020 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "RLMUUID_Private.hpp"
+#import <realm/uuid.hpp>
+
+@implementation NSUUID (RLMUUIDSupport)
+
+- (instancetype)initWithRealmUUID:(realm::UUID)rUuid {
+    self = [self initWithUUIDBytes: rUuid.to_bytes().data()];
+    return self;
+}
+
+- (realm::UUID)uuidValue {
```

Comment:
This needs to be rlm_uuidValue; extension methods on standard library types need to have our library prefix on them to avoid clashing.

Reply:
Fixed